ndsl/stencils/testing/serialbox_to_netcdf.py

```diff
@@ -32,6 +32,12 @@ def get_parser():
         type=str,
         help="[Optional] Give the name of the data, will default to Generator_rankX",
     )
+    parser.add_argument(
```

Comment:
We are working on an extension to Serialbox that will deal with this Fortran side.